commands/import.go

```diff
@@ -159,20 +157,21 @@ func setRolling(dbExists bool, dbIsRolling bool, dbCurrChunk int, dbTotalChunks
 		}
 	}
 
-	// already existing db is converted to rolling if it wasn't already
-	// or if the user specified that a new database is rolling
-	cfg.Rolling = dbExists || userIsRolling
+	cfg.Rolling = dbIsRolling || userIsRolling
 
 	// validate chunk size
-	if (cfg.CurrentChunk < 0 ||
-		cfg.CurrentChunk >= cfg.TotalChunks) {
+	if cfg.CurrentChunk < 0 ||
+		cfg.CurrentChunk >= cfg.TotalChunks {
 		return cfg, fmt.Errorf(
 			"\n\t[!] Current chunk number [ %d ] must be 0 or greater and less than the total number of chunks [ %d ]",
 			cfg.CurrentChunk,
 			cfg.TotalChunks,
 		)
 	}
 
+	// preserve the default chunks setting (even though we don't use it after this currently)
+	cfg.DefaultChunks = cfgDefaultChunks
```

Comment:
This was set outside of this function, cluttering up the main function. I moved it here since this function is the only one which writes to the config. However, I had to change the test cases to match this.